src/DropdownButton/Option.tsx

```diff
@@ -21,7 +21,7 @@ const propTypes = {
 
 export default class Option extends React.PureComponent<Props> {
   static propTypes = propTypes;
-  render() {
+  render(): never {
```

Comment:
Whoa! TIL

Reply:
As a maybe interesting side note, the compiler correctly infers `never` as the return type of a function that always throws
<img width="657" alt="Screen Shot 2019-07-01 at 12 36 47 AM" src="https://user-images.githubusercontent.com/32328921/60418790-ad84b300-9b98-11e9-82d8-3d42c6f84073.png">

The compiler usually does a good job of inferring the return type, but it seems that it infers `void` as the return type for `React.Component`s that unconditionally throw. `JSX.ReactElement` doesn't like being `void`.